lib/active_merchant/billing/gateways/visanet_peru.rb

```diff
@@ -77,7 +77,7 @@ def scrub(transcript)
         transcript.
           gsub(%r((Authorization: Basic )\w+), '\1[FILTERED]').
           gsub(%r((\"cardNumber\\\":\\\")\d+), '\1[FILTERED]').
-          gsub(%r((\"cvv2Code\\\":)\d+), '\1[FILTERED]')
+          gsub(%r((\"cvv2Code\\\":\\\")\d+), '\1[FILTERED]')
```

Comment:
I should have said all of the *unit* tests were working for me on master in my initial reply. :)